### PR TITLE
[Hub Generated] Review request for Microsoft.Resources to add version stable/2020-01-01

### PR DESCRIPTION
**Pull request by dbbooher**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/dbbooher/azure-rest-api-specs/dev-resources-Microsoft.Resources-2020-01-01...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/resources/resource-manager/readme.md [hunk 1]
```diff
@@ -57,9 +57,18 @@ tag: package-managedapplications-2018-06
...
 
+
+### Tag: package-2020-01
+
+These settings apply only when `--tag=package-2020-01` is specified on the command line.
+
+```yaml $(tag) == 'package-2020-01'
+input-file:
+  - Microsoft.Resources/stable/2020-01-01/subscriptions.json
+```
```
Comment (Tiano2017) [inline specification/resources/resource-manager/readme.md]: blank line after this.

**Review by Tiano2017 — APPROVED** [review]

File: specification/resources/resource-manager/readme.md [hunk 2]
```diff
@@ -57,9 +57,18 @@ tag: package-managedapplications-2018-06
 ```
 
 ``` yaml $(package-deploymentscripts)
-tag: package-deploymentscripts-2019-10-preview
+tag: package-2020-01
```
Comment (yungezz) [inline specification/resources/resource-manager/readme.md]: the tag naming is not consistent with previous one such as `package-subscription-2019-10-preview`, `package-resources-2019-10' etc.

Comment (dbbooher) [reply specification/resources/resource-manager/readme.md]: This was auto-generated for me by the OpenAPI Hub tool, but it looks like it dropped a new package tag in the middle of an existing one. If you can help me understand what these tags do and why they're added, I can try to straighten this out.

Comment (dbbooher) [reply specification/resources/resource-manager/readme.md]: I think I fixed it. Let me know if this looks better

**Comment by dbbooher** [comment]
@yungezz I think I've fixed the readme per your last comment. Can you review and merge?

**Review by yungezz — APPROVED** [review]

**Pull request merged**